scripts/ci/release_trigger_guard.py

```
@@ -293,6 +307,78 @@ def main() -> int:
                         except RuntimeError as exc:
                             warnings.append(f"Failed to inspect tagger metadata for `{args.release_tag}`: {exc}")
 
+                    # --- CI green gate (blocking) ---
+                    if tag_commit:
+                        ci_check_proc = subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)